bwc-test/build.gradle

```diff
@@ -70,19 +67,37 @@ repositories {
 }
 
 dependencies {
+    testImplementation 'com.google.guava:guava:30.0-jre'
     testImplementation "org.opensearch.test:framework:${opensearch_version}"
+    testImplementation 'org.apache.logging.log4j:log4j-core:2.17.1'
 }
 
-String bwcVersion = "2.4.0.0";
+loggerUsageCheck.enabled = false
+testingConventions.enabled = false
+validateNebulaPom.enabled = false
+
+String previousVersion = System.getProperty("bwc.version.previous", "2.4.0.0")
+String nextVersion = System.getProperty("bwc.version.next", "3.0.0.0")
```

Comment:
Gotcha. That all sounds good then. Thank you for clarifying, I just know I have seen you do some fancy parsing stuff in the past and was not sure if this would be a good opportunity to do likewise.